app/mailers/devise_custom_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Mailer Overrides for Devise default emails
+class DeviseCustomMailer < Devise::Mailer
+  include Devise::Controllers::UrlHelpers # Optional. eg. `confirmation_url`
+  default template_path: 'devise/mailer' # to make sure that your mailer uses the devise views
+
+  def confirmation_instructions(record, token, opts = {})
+    @greeting_name = record.greeting_name || ''
```

Comment:
Can we remove `|| ''` as it makes `@greeting_name` truthy and displays "Hi ," in the email?

<img width="1679" alt="image" src="https://user-images.githubusercontent.com/7039523/91924292-ebe39980-ec97-11ea-8530-a92a021646b0.png">